libs/sdk-ui-kit/src/Dialog/ConversionTouchPointDialogBase.tsx

```diff
@@ -0,0 +1,189 @@
+// (C) 2021 GoodData Corporation
+import React, { useEffect, useState } from "react";
+import { useIntl } from "react-intl";
+import { useHubspotForm } from "@aaronhayes/react-use-hubspot-form";
+import { HubSpotFormLocale } from "@aaronhayes/react-use-hubspot-form/build/main/hubspot-form-locale.enum";
+import { LoadingComponent } from "@gooddata/sdk-ui";
+
+import { Button } from "../Button";
+
+declare global {
+    interface Window {
+        jQuery: object;
+        hbspt: undefined | { isSuccessMessageShow: boolean };
+    }
+}
+
+/**
+ * @internal
+ */
+export interface IHubspotFormField {
+    /**
+     * Form Field Name
+     */
+    name: string;
+
+    /**
+     * Form Field Value
+     */
+    value: string | any;
+
+    /**
+     * Click action
+     */
+    click: () => void;
+}
+
+/**
+ * @public
+ */
+export interface IHubspotFormValue {
+    [key: string]: string | number | boolean;
+}
+
+/**
+ * @public
+ */
+export interface IConversionTouchPointDialogBaseProps {
+    /**
+     * Hubspot Portal Id
+     */
+    hubspotPortalId: string;
+
+    /**
+     * Hubspot Form Id
+     */
+    hubspotFormId: string;
```

Comment:
Please notify User how to get hubspotPortalId, hubspotFormId like [here](https://github.com/gooddata/gooddata-ui-sdk/pull/1746#:~:text=gooddata-ui-sdk/libs/sdk-ui-ext/src/internal/dashboardEmbedding/types.ts)

Reply:
We don't have the link to help the users get their portal id and form id. They have to find it.